x-pack/test/functional/apps/observability/alerts/index.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import querystring from 'querystring';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+// Based on the x-pack/test/functional/es_archives/observability/alerts archive.
+const DATE_WITH_DATA = {
+  rangeFrom: '2021-08-23T13:36:22.109Z',
+  rangeTo: '2021-08-25T13:36:22.109Z',
+};
+
+export default ({ getPageObjects, getService }: FtrProviderContext) => {
+  const esArchiver = getService('esArchiver');
+
+  describe('Observability alerts', function () {
+    this.tags('includeFirefox');
+
+    const pageObjects = getPageObjects(['common']);
+    const testSubjects = getService('testSubjects');
+
+    before(async () => {
+      await esArchiver.load('x-pack/test/functional/es_archives/observability/alerts');
+      await pageObjects.common.navigateToUrlWithBrowserHistory(
+        'observability',
+        '/alerts',
+        `?${querystring.stringify(DATE_WITH_DATA)}`
+      );
+    });
+
+    after(async () => {
+      await esArchiver.unload('x-pack/test/functional/es_archives/observability/alerts');
+    });
+
+    describe('Alerts table', () => {
+      it('Renders the table', async () => {
+        await testSubjects.existOrFail('events-viewer-panel');
```

Comment:
Would it be a good idea to keep a file of subject selector constants?

Reply:
We should create a custom Observability Alerting provider moving forward. Here's an example one: https://github.com/elastic/kibana/blob/master/x-pack/test/functional/services/logs_ui/log_stream.ts, these services let us create functions to perform little selector shortcuts / common interactions etc. I haven't added it for this first addition, but as we'll be adding a lot more functional tests it can come soon. It would be somewhat related to: https://github.com/elastic/kibana/issues/110627

Reply:
Yeah, I sneakily turned the "set up suites with different permission combinations" task we talked about into actually creating a testing framework service. :innocent: I like how the ml team set up a substructure for their page objects/services within their provider.

Reply:
Sounds good!